pandas/core/indexing.py

```diff
@@ -145,9 +149,6 @@ def __getitem__(self, arg):
         return arg
 
 
-IndexSlice = _IndexSlice()
```

Comment:
Wondering if we should keep a `_IndexSlice = IndexSlice` here for backwards compatibility (I know this is private, but you never know ..)

Reply:
From a quick github search I don't find much public use, but for example the stubs use the current import to get the class for type annotations

Reply:
> From a quick github search I don't find much public use, but for example the stubs use the current import to get the class for type annotations

hmm. I'll look in more detail (probs tomorrow now) before making the changes since if the class is needed for type annotations then perhaps the class should be exposed in pandas.api.typing and the repr should point to that.